go-controller/pkg/crd/egressservice/v1/apis/clientset/versioned/fake/clientset_generated.go

```diff
@@ -73,6 +78,38 @@ func (c *Clientset) Tracker() testing.ObjectTracker {
 	return c.tracker
 }
 
+// NewClientset returns a clientset that will respond with the provided objects.
```

Comment:
I think we should start using `NewClientset` instead of `NewSimpleClientset` in our tests.
This should probably be a refactor effort in the future.

Reply:
agree on the future part :stuck_out_tongue: